### DISCUSSION TITLE
How are primary-Category objects and definitions initialised when moving up the environments?

### DISCUSSION THREAD

**Discussion by joffinjoy** [Sunbird-inQuiry/Community#19] [Q&A] [answered]
In my local, we used object creation and definition creation curls (Taxonomy service) for creating primary categories. Is that still the recommended way to create primary categories when moving up the environments like staging and pre-prod etc?

cc: @aks30, @vijiurs

**Comment by krgauraw** [comment, marked as answer]
@joffinjoy In all env, we should use the object category & object category definition api's provided by taxonomy-service to populate the primary category objects and its definition. We do have some readymade curls [here](https://github.com/Sunbird-inQuiry/inquiry-api-service/tree/release-5.0.0/scripts/definition-scripts) . You can use them.
cc: @pallakartheekreddy @aks30 @vijiurs @surendrasinghs

**Reply by joffinjoy** [reply]
Thanks @krgauraw
